src/App/components/Resource/components/Examples/index.js

```diff
@@ -0,0 +1,190 @@
+import React, {Component} from 'react'
+import {map, includes, size, uniqueId} from 'lodash'
+import {
+  smallContainerWidth,
+  mediumContainerWidth,
+  largeContainerWidth,
+  xlargeContainerWidth,
+  xxlargeContainerWidth,
+} from 'package/utils/hardCodedSizes'
+import Heading from 'package/components/Heading'
+import Button from 'package/components/Button'
+
+const containerWidthActions = [
+  {
+    label: 'Small',
+    containerWidth: smallContainerWidth,
+  },
+  {
+    label: 'Medium',
+    containerWidth: mediumContainerWidth,
+  },
+  {
+    label: 'Large',
+    containerWidth: largeContainerWidth,
+  },
+  {
+    label: 'XLarge',
+    containerWidth: xlargeContainerWidth,
+  },
+  {
+    label: 'XXLarge',
+    containerWidth: xxlargeContainerWidth,
+  },
+]
+
+const containerBackgroundActions = [
+  {
+    label: 'Dark',
+    containerBackground: 'dark',
+  },
+  {
+    label: 'Light',
+    containerBackground: 'light',
+  },
+]
+
+const containerBoxSizing = 8
+
+export const optOuts = [
+  'containerWidth',
+  'containerBackground',
+  'types',
+]
+
+class Examples extends Component {
+
+  state = {
+    containerWidth: smallContainerWidth,
+    containerBackground: 'dark',
+    examples: false,
+  }
+
+  componentWillMount = () => {
+    this.handleCreateExamples()
+  }
+
+  handleContainerWidthChange = (containerWidth) => {
+    this.setState({
+      containerWidth,
+    })
+  }
+
+  handleBackgroundChange = (containerBackground) => {
+    this.setState({
+      containerBackground,
+    })
+  }
+
+  handleCreateExamples = () => {
+    this.setState({
+      examples: this.props.createExamples(),
+      renderId: uniqueId(),
+    })
+  }
+
+  render() {
+    const {containerWidth, containerBackground, examples, renderId} = this.state
+    const {optOut} = this.props
+
+    return (
+      <section>
+
+        {size(optOut) < size(optOuts)
+          ? <div className='bg-base-secondary pa3 br2 mb4'>
```

Comment:
Thanks @ijones16! Yes I tried that but was getting errors because the inner elements were still rendering; I don't understand it fully yet - something with the way React children work, but `Maybe` needs some work to act just like a ternary.